components/metrics/BUILD.gn

```diff
@@ -474,3 +472,34 @@ if (!use_cobalt_customizations) {
     ]
   }
 }
+
+if (use_cobalt_customizations) {
+  target(gtest_target_type, "components_metrics_tests") {
```

Comment:
Are you able to use the metrics_unittests target above along with `//testing/test.gni`? Or is the testing setup very different (i.e. can't rely on `run_all_unittests.cc`/`test_support`)?

Reply:
I started trying to just use `metrics_unittests`, but quickly ran into issues. As you mentioned, we just don't use the `run_all_unittests.cc` code... and several of the test libraries were missing. After lots of refactoring/moving deps/sources around, I realized what I had was so far from the original that I might as well just create my own target. Which is much cleaner, IMO.

Reply:
Makes sense